bluezero/broadcaster.py

```diff
@@ -40,9 +40,13 @@ def add_manufacturer_data(self, manufacturer, data):
         Add manufacturer information to be used in beacon message
         :param manufacturer: Use numbers from Bluetooth SIG
         https://www.bluetooth.com/specifications/assigned-numbers/16-bit-UUIDs-for-Members
-        :param data: Data to be sent (Limit of ??)
+
+        :param data: Data to be sent (Limit of 23bytes)
```

Comment:
`bytearray.fromhex(23*"FF")` doesn't feel right to me I would prefer something like: `b'\xff' * 23` if you want to keep the line length manageable.